user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,81 @@ Please note the following regarding suggestion events:
+
+Feedback can be provided via the ![Feedback](~/user-guide/images/Feedback_Column.png) column, which is visible by default in the *Anomalies* and *Suggestion events* tabs of the Alarm Console. To make the column visible in other tabs that list either alarms or suggestion events, right-click the column headers and select *Add or remove columns* > *Actions* > *Feedback*<!--RN 39640-->.
+
+> [!TIP]
+> See also: [*Improving anomaly detection using feedback* tutorial](xref:Anomaly_Feedback_Tutorial)
+
+### Prerequisites
+
+- Behavioral anomaly detection is enabled, and all its [prerequisites](xref:Advanced_analytics_trending) are met.
+
+- [Storage as a Service (STaaS)](xref:STaaS) or an [indexing database](xref:Supported_system_data_storage_architectures).
+
+- The [time to live](xref:Specifying_TTL_overrides) for trending is at least 1 day for real-time trending and 1 month for minute records.
+
+### Providing feedback
+
+<!--RN 39640-->
+
+To provide feedback:
+
+1. In the Alarm Console, open the *Anomalies* or *Suggestion events* tab. See [Navigating the light bulb menu](xref:Light_Bulb_Feature#navigating-the-light-bulb-menu) and [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+1. Verify that the ![Feedback](~/user-guide/images/Feedback_Column.png) column is visible. If it is not:
+
+   - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+     This step might be necessary if you have recently upgraded from a version of DataMiner that did not include the Feedback column to one that does (DataMiner 10.4.11/10.5.0 or higher). Since Cube saves your open tabs along with their column layout, the new column will not automatically appear in tabs that were open before the upgrade.
+
+   - Ensure you are using DataMiner version 10.4.11/10.5.0 or higher, or confirm that the *AnomalyFeedback* option is included in the *SoftLaunchOptions.xml* file.
+
+   - If possible, manually add the column by right-clicking the column headers and selecting *Add or remove columns* > *Actions* > *Feedback*.
+
+1. Hover the mouse pointer over a suggestion event or alarm to view the ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) and ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons.
+
+   - If you approve of the anomaly and want to receive similar ones in the future, click the thumbs-up icon in the feedback column. Alternatively, you can right-click the suggestion event/alarm and select *Like*.
+
+   - If you disapprove of the anomaly and do not wish to be notified of similar events in the future, click the thumbs-down icon. Alternatively, you can right-click the suggestion event/alarm and select *Dislike*.
```

Comment:
I think it might be better if we avoid "approve of the anomaly" and "disapprove of the anomaly", as this sounds a bit odd: the anomaly in itself may not be what the user approves of, since it might point at something going wrong in their system.
```suggestion
   - If you want to be notified of similar anomalies in the future, click the thumbs-up icon in the feedback column. Alternatively, you can right-click the suggestion event/alarm and select *Like*.

   - If you do not want to be notified of similar events in the future, click the thumbs-down icon. Alternatively, you can right-click the suggestion event/alarm and select *Dislike*.
```